openpos-service/src/main/java/org/jumpmind/pos/service/strategy/RemoteOnlyStrategy.java

```
@@ -129,6 +133,9 @@ private HttpHeaders getHeaders(ProfileConfig profileConfig, String profileId) {
     }
 
     private Object invokeProfile(String profileId, EndpointInvocationContext endpointInvocationContext) throws ResourceAccessException {
+        if (statusMonitor.isOffline(profileId)) {
```

Fail fast if we already know the profile is offline